.travis.yml

```diff
@@ -1,7 +1,4 @@
 language: node_js
 node_js:
-    - "5"
-    - "5.1"
-    - "4.2"
-    - "4.1"
-    - "4.0"
+    - node
+    - 8.3
```

Comment:
I wonder if updating this travis file like this has an effect on the github tests

Reply:
i needed to update because i uses spread operator so tests crashed every time

Reply:
ah I see, that is worrying - perhaps we need to revert this and not use deconstruction in that one place - as it will crash bondagejs on some older browsers

Reply:
Can we revert the changes to this file now? Tests should pass as we removed the spread operator